gen_partition.py

```diff
@@ -151,8 +151,7 @@ def parse_disk_entry(disk_entry):
          print (str(e))
          usage()
 
-def generate_ptool_xml (disk_params, partition_entries_dict, output_xml):
-   print("Generating ptool XML %s" %(output_xml))
+def generate_emmc_xml (disk_params, partition_entries_dict, output_xml):
```

Comment:
A bit confusing that this was renamed to have emmc in the name but it is also used with nvme now.

Reply:
That would reflect more how we use it, at least.